components/script/dom/window.rs

```diff
@@ -1374,35 +1374,39 @@ impl Window {
         issued_reflow
     }
 
+    pub fn layout_reflow(&self, query_msg: QuerMsg) -> bool {
```

Comment:
Let's call this `layout_query` instead. Also, change `QuerMsg` to `QueryMsg` so this commit builds by itself.